README.md

```diff
@@ -1,20 +1,49 @@
-# elrrdatasync
+# elrrdatasync API
 ELRR services which aid in the streaming (Kafka streams) of data from the staging database to the ELRR database.
 
-# Requirements
-For building and running the elrrdatasync you need:
-- JDK 1.8
+Setup elrrdatasync first.
+
+# Dependencies
+- Java JDK 17
+- git
 - Maven 3
-# Build the application
-- mvn clean install -Dmaven.test.skip=false
-# Deploying the application on Docker 
```

Comment:
this is mostly still relevant docs. I would have assumed you would add a section later on for local deployments, or just replace the "Running Application Locally" section.